.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

we should still allow blank issues just in case
```suggestion
blank_issues_enabled: true
```